### PR TITLE
Creating Data Model

### PR DESCRIPTION
**Pull request by Carshy**
In the current project, I used migration files to build my schema based on the erd diagram provided for the project.

### CODE REVIEW

File: db/migrate/20221028081629_create_coments.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateComents < ActiveRecord::Migration[7.0]
+  def change
+    create_table :coments do |t|
```
Comment (DammyShittu) [inline db/migrate/20221028081629_create_coments.rb]: - [ ] I think this table name should be `comments`, not `coments`. Kindly fix this table name.

- [ ] Remember to rename the related model and file name as well: **`Comment`**

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Carshy,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Carshy** [comment]
Changes made as requested. @DammyShittu thanks for your review it was really helpful

File: app/models/post.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :coments
```
Comment (CollinsTatang) [inline app/models/post.rb:3]: ### **OPTIONAL**

- [ ] Please fix this typo error for `comments`

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/post.rb (hunk 2 above)
Comment (Carshy) [reply app/models/post.rb:3]: @CollinsTatang Thanks for pointing this out I will be  fixing it right away!

**Pull request merged**